apps/website/src/global.css

```diff
@@ -154,12 +154,6 @@
       var(--qwikui-slate-900)
     );
   }
```

Comment:
Add this back, otherwise we get horizontal overflow on the code examples.

Reply:
![image](https://github.com/qwikifiers/qwik-ui/assets/104264123/0a9ecd19-ca80-431b-a938-f0ba0845c1c3)